react/components/ProductAssemblyOptions/InputValue/useInputValue.tsx

```diff
@@ -5,7 +5,7 @@ interface OnChangeParams {
 }
 
 export default function useInputValue(inputValueInfo: InputValue): [string, (param: OnChangeParams) => void] {
-  const { path, valuesOfInputValues } = useProductAssemblyGroupState() as AssemblyOptionGroupType
+  const { path, valuesOfInputValues } = useProductAssemblyGroupState() as AssemblyOptionGroupState
```

Comment:
Just for curiosity, but why do you need to use `as AssemblyOptionGroupState` in this call since this is coming from a custom hook? Is the return type for this hook not supposed to be `AssemblyOptionGroupState`?